algo_marg_settings.asciipb

```diff
@@ -25,3 +25,12 @@ init_num_clusters: 3
 # (NOTE: 3 is the recommended value in most cases, please change it only if you
 #  know what you're doing.)
 neal8_n_aux: 3
+
+# Split and Merge number of restricted GS scans for each MH step.
```

Comment:
I'd put a title to make it clear that the following parameters are only used with the S&M algorithm